.travis.yml

```diff
@@ -19,6 +22,10 @@ matrix:
       env: deps="low"
     - php: 7.0
       env: xdebug="yes"
+      php: 7.1
+      env: xdebug="yes"
```

Comment:
using Xdebug for the code coverage on one job is enough, I would just remove it for the new entries and keep it for the PHP 7.0 job